test/python/synthesis/test_synthesis.py

```diff
@@ -986,15 +995,23 @@ def test_cnot_rxx_decompose(self):
         """Verify CNOT decomposition into RXX gate is correct"""
         cnot = Operator(CXGate())
         decomps = [
-            cnot_rxx_decompose(),
-            cnot_rxx_decompose(plus_ry=True, plus_rxx=True),
-            cnot_rxx_decompose(plus_ry=True, plus_rxx=False),
-            cnot_rxx_decompose(plus_ry=False, plus_rxx=True),
-            cnot_rxx_decompose(plus_ry=False, plus_rxx=False),
+            _cnot_rxx_decompose(),
```

Comment:
Now that this is a private method, do we still need to test it or is it tested enough in the context of the composer? If yes, I'd think we can skip testing this